src/extensions/sessionrecording.js

```diff
@@ -142,7 +142,6 @@ export class SessionRecording {
 
                 this._updateWindowAndSessionIds(event)
 
-                delete event.data?.['plugin'] // gets sent on every console.log and is unnecessary
```

Comment:
I ended up needing this plugin param on the front-end. It lets you differentiate between types of plugin data when parsing the payloads.